cmd/podman/create.go

```diff
@@ -174,11 +174,14 @@ func createCmd(c *cli.Context) error {
 	}
 	defer runtime.Shutdown(false)
 
-	imageName, _, data, err := imageData(c, runtime, c.Args()[0])
+	rtc := runtime.GetConfig()
+
+	newImage, err := runtime.ImageRuntime().New(c.Args()[0], rtc.SignaturePolicyPath, "", image.CreateStdoutWriter(), nil, image.SigningOptions{})
```

Comment:
Should not need CreateStdoutWriter here - should just be able to use os.Stderr (we prefer stderr for progress bars)

Reply:
fixed